bin/pdfexport.py

```diff
@@ -0,0 +1,38 @@
+import re
+import sys
+import requests
+
+#
+# Standalone script to batch download all plans from an EasyDMP instance
+#
+
+if len(sys.argv) < 4:
+    print('Usage: pdfexport <endpoint> <token> <directory> [from_date] [to_date]')
+    print('Time format: 2019-09-30T23:59:59.00Z')
+    exit(0)
+
+endpoint = sys.argv[1]
+token = sys.argv[2]
+dir = sys.argv[3]
+
+from_date = sys.argv[4] if len(sys.argv) >= 5 else None
+to_date = sys.argv[5] if len(sys.argv) >= 6 else None
+
```

Comment:
Why not optparse or argparse? Having it in a function would alos be handy.

Reply:
@hmpf To have minimal reqs, since this might be used standalone without virtualenv, for example.